ddtrace/llmobs/_trace_processor.py

```diff
@@ -56,11 +57,16 @@ def process_trace(self, trace: List[Span]) -> Optional[List[Span]]:
 
     def submit_llmobs_span(self, span: Span) -> None:
         """Generate and submit an LLMObs span event to be sent to LLMObs."""
+        span_event = None
         try:
             span_event = self._llmobs_span_event(span)
             self._span_writer.enqueue(span_event)
         except (KeyError, TypeError):
             log.error("Error generating LLMObs span event for span %s, likely due to malformed span", span)
+        finally:
+            if span_event and self._evaluators:
+                for evaluator in self._evaluators:
+                    evaluator.enqueue(span_event)
```

Comment:
```suggestion
            if not span_event:
                return
            for evaluator in self._evaluators:
                evaluator.enqueue(span_event)
```
I assume `self._evaluators` is always empty if not configured, so we can directly go into the for loop instead of checking